js/modules/types/attachment.js

```diff
@@ -29,19 +29,17 @@ const { migrateDataToFileSystem } = require('./attachment/migrate_data_to_file_s
 //   schemaVersion: integer
 // }
 
-// Returns true if `rawAttachment` is a valid attachment based on our (limited)
-// criteria. Over time, we can expand this definition to become more narrow:
+// Returns true if `rawAttachment` is a valid attachment based on our current schema.
+// Over time, we can expand this definition to become more narrow, e.g. require certain
+// fields, etc.
 exports.isValid = (rawAttachment) => {
   // NOTE: We cannot use `_.isPlainObject` because `rawAttachment` is
   // deserialized by protobuf:
   if (!rawAttachment) {
     return false;
   }
 
-  const hasValidContentType = isString(rawAttachment.contentType);
-  const hasValidFileName =
-    isString(rawAttachment.fileName) || rawAttachment.fileName === null;
-  return hasValidContentType && hasValidFileName;
+  return true;
```

Comment:
Temporarily removed this check as you found `fileName === undefined`, @scottnonnenberg-signal.